pkgs/development/tools/knightos/z80e/default.nix

```diff
@@ -11,6 +11,8 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-FQMYHxKxHEP+x98JbGyjaM0OL8QK/p3epsAWvQkv6bc=";
   };
 
+  strictDeps = true;
```

Comment:
Currently fails, since `knightos-scas` is in `nativeBuildInputs`, the headers/libraries are not visible during the build.

Reply:
Right. I'll remove that. I can't test this build due to SDL2 failing to cross-compile to raspberryPi